extension/README.md

```diff
@@ -1,2 +1,34 @@
-# vscode-xcomponent
-visual studio code extension for xcomponent
+# xcomponent
+
+This extension provides a development environment for [XComponent](http://www.xcomponent.com/).
```

Comment:
this should be on the higher level :)

on this level we need a more low level explanation (low level != longer)